### ISSUE TITLE
Feature incompatible

### ISSUE THREAD

**Issue by flankerhqd** [secure-software-engineering/CodeInspect#128]
Some features, like "if-else generation" will produce errors and make packing process unusable, thus make debugging usable. I'll explain it in a github issue.

**Comment by MarcMil** [comment]
Could you elaborate on what apk you used and what kind of Deobfuscation features you had activated (i.e. only the Block-Generator)?
It is known that the Block-Generator in the current state is capable of generating semantic wrong code (due to Soot's Dava), however, it should not result in a syntactic error.

**Comment by flankerhqd** [comment]
Yes, I've tried block-generator, however this feature before produces errors, now it directly cleans all jimple content.
